projects/step-frontend/src/lib/modules/timeseries/execution-page/time-series-keywords.context.ts

```diff
@@ -14,12 +14,16 @@ export class TimeSeriesKeywordsContext {
   private keywordToggleSubject = new Subject<KeywordSelection>();
   private selectAllSubject = new BehaviorSubject<boolean>(true);
 
-  private colorsPool: TimeseriesColorsPool;
+  private readonly colorsPool: TimeseriesColorsPool;
 
   constructor(colorsPool: TimeseriesColorsPool) {
     this.colorsPool = colorsPool;
   }
```

Comment:
From line 17 to 21 - it can be shorten as:
```
constructor(public readonly colorsPool: TimeseriesColorsPool) {
}
```
The public readonly field will be automatically created and will it take value from the class constructor